Sources/SPTDataLoaderServiceSessionSelector.m

```diff
@@ -24,9 +24,13 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 NS_ASSUME_NONNULL_BEGIN
 
+static NSString *const SPTDataLoaderBackgroundSessionConfigurationIdentifier =
+    @"SPTDataLoaderBackgroundSessionConfigurationIdentifier";
+
 @interface SPTDataLoaderServiceDefaultSessionSelector ()
 
 @property (nonatomic, strong, readonly) NSURLSessionConfiguration *configuration;
+@property (nonatomic, strong, readonly) NSURLSession *backgroundSession;
```

Comment:
is that needed?

Reply:
I think it makes sense to declare a property if we're going to use dot notation for accessing it. But up to you, I can remove it as well.

Reply:
Not a super expert for the Obj-C language so I might be wrong.
wouldn't `self.backgroundSession` call the `backgroundSession` method that was added?

That is what `self.waitingSession` does and there is no property for that. That is why I am asking

Reply:
Yes, it will work the same way. Just a code clarity thing.

Reply:
It is not super important but in my opinion to conform to the same style in the file, either this one is not introduced or the other ones are added also.